core/base/document.php

```diff
@@ -1093,6 +1100,30 @@ public function get_last_edited() {
 		return $last_edited;
 	}
 
+	public function add_repeater_row( $control_id, $item ) {
```

Comment:
It's for page_settings only - move it to the `kit` document.

Reply:
Handled: https://github.com/elementor/elementor/pull/11539/commits/933bf1308e10360c3ef6d39a852fc491fb33fb87